orderer/consensus/smartbft/util_network_test.go

```diff
@@ -186,6 +192,8 @@ func (n *Node) Restart() error {
 	}
 	n.Chain = newChain
 	n.Chain.Start()
+	n.IsConnectedToNetwork = true
+	n.IsStarted = true
```

Comment:
why are you calling `n.Chain.Halt()` when you restart?

Reply:
It seemed to me that this was necessary, but in fact it is not required.